helpers.py

```diff
@@ -21,9 +21,78 @@
 import subprocess
 import time
 
-from fabric import Config
-from fabric import Connection
-from paramiko import SSHException
+
+class Result:
+    def __init__(self, stdout, stderr, exited):
+        self.stdout = stdout
+        self.stderr = stderr
+        self.exited = exited
+        self.return_code = exited
+
+
+class Connection:
+    def __init__(self, host, user, port, connect_timeout, key_filename=None):
+        self.host = host
+        self.user = user
+        self.port = port
+        self.connect_timeout = connect_timeout
+        self.key_filename = key_filename
+        self.key = _prepare_key_arg(key_filename)
+
+    def __enter__(self):
+        return self
+
+    def get_ssh_command(self):
+        return f"ssh {self.key} -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o ConnectTimeout={self.connect_timeout} -p {self.port} {self.user}@{self.host}"
+
+    def run(self, command, warn=False, hide=True, echo=False, popen=False):
+        cmd = f"{self.get_ssh_command()} '{command}'"
+
+        # Allows the function to run full ssh and scp commands
+        if not command.startswith("ssh") and not command.startswith("scp"):
+            command = cmd
```

Comment:
I think that it would read better with:
```suggestion
         # Allows the function to run full ssh and scp commands
         if not command.startswith("ssh") and not command.startswith("scp"):
             command = f"{self.get_ssh_command()} '{command}'"
```